src/sdk/index.ts

```diff
@@ -605,6 +594,12 @@ export class FhenixClientSync extends FhenixClientBase {
   ): Promise<FhenixClientSync> {
     isPlainObject(params);
 
+    if (params.skipPubKeyFetch !== false) {
```

Comment:
This introduces a breaking change to `FhenixClientSync`. Should be `params.skipPubKeyFetch === true` so that users aren't forced to set `skipPubKeyFetch` in their params.

Reply:
why? I wanted the default to fetch the pubkey, as it did previously

Reply:
The default is still staying the same, but `params.skipPubKeyFetch?: boolean` defaults to `undefined` which will enter this conditional